dGame/dMission/MissionTask.cpp

```diff
@@ -352,9 +343,16 @@ void MissionTask::Progress(int32_t value, LWOOBJID associate, const std::string&
 		if (info->target != gameID) {
 			break;
 		}
-
+		// Lego Universe is a great game with very easy to understand mission progression.
+		// If we are progressing a mission for shooting gallery we need to set its
+		// completion to 1 and not the targetValue beacuse that makes sense
+		// compared to all other missions and tasks :)
+		if(info->targetGroup == targets && value >= info->targetValue && GetMission()->IsMission() && info->target == 1864 && info->targetGroup == "performact_score") {
```

Comment:
Im not sure if this whole if statement is needed, however i am being more safe than sorry here.  Something needs to be done to the charxml to fix players who are in a state where this value is larger than it should be.